File: Q02.java
import java.util.Scanner;

class segundaClasse {
    public static void main(String[] args) {
        Scanner myObj = new Scanner(System.in);
        System.out.println("Digite um número: ");

        int numero = myObj.nextInt();

        if (numero % 2 == 0){
            System.out.println("O número é par.");
        }
        else if (numero % 2 != 0) {
            System.out.println("O número é ímpar.");
        }
    }
}

File: Q02.hs
main :: IO ()
main = do
    putStrLn "Digite um inteiro:"
    input <- getLine
    let inteiro = read input :: Int

    if inteiro `mod` 2 == 0 
    then do putStrLn "O número é par."
    else if inteiro `mod` 2 == 1 
    then do putStrLn "O número é ímpar."
    else do putStrLn "Inválido."